projects/core/src/routing/configurable-routes/configurable-routes.service.ts

```diff
@@ -125,26 +122,26 @@ export class ConfigurableRoutesService {
   private translateRoutePath(
     route: Route,
     routesTranslations: RoutesTranslations
-  ): Route[] {
-    return this.getTranslatedPaths(route, 'cxPath', routesTranslations).map(
-      translatedPath => {
-        return { ...route, path: translatedPath };
-      }
-    );
+  ): Route {
+    const paths = this.getTranslatedPaths(route, 'cxPath', routesTranslations);
+    const matcher = paths.length
+      ? UrlMatcherFactory.getPathsUrlMatcher(paths)
+      : UrlMatcherFactory.getFalsyUrlMatcher(); // if there are no configured paths for this route, never match this route
```

Comment:
Isn't `FalsyUrlMatcher` little bit of overengineering? 
Don't get me wrong, the solution is a clever one, but do we really need that route in configuration at all? It will be never matched, but routing engine will hit it very often when resolving navigation. It will also be visible in some debuggin tools, without any indication, that it is a dead entry,

Reply:
We have 2 options to disable a route:
- remove it from config
- set a falsy matcher.
The second option lets us re-enable the route when it's enabled in the config for different language or base site.